README.md

```diff
@@ -57,6 +57,21 @@ A diagram of this running 5 steps on a single sample image may help explain how
 ## Installation
 
 This library conveniently works in the browser, in Node, and on the command line (CLI).
+To install the app, run the following commands in a command line:
+
+* `git clone https://github.com/vaibhavmatta/image-sequencer`
```

Comment:
lets not be so specific with the https clone link only, users can also do it using ssh.

Reply:
I just wanted to give them the exact line of code so that they do not waste der time. I will add an optional line for ssh as well.